src/teleop/gui/src/components/AutonRoverMap.vue

```diff
@@ -33,6 +35,30 @@
         </l-tooltip>
       </l-marker>
 
+      <!-- Search Path Icons -->
+      <l-marker
+        v-for="(search_path_point, index) in searchPathPoints"
+        :key="index"
+        :lat-lng="search_path_point.latLng"
+        :icon="searchPathIcon"
+      >
+        <l-tooltip :options="{ permanent: 'true', direction: 'top' }">Search Path {{
+          index
+        }}</l-tooltip>
+      </l-marker>
+
+      <!-- Gate Path Icons-->
+      <l-marker
+        v-for="(gate_path_point, index) in gatePathPoints"
+        :key="index"
+        :lat-lng="gate_path_point.latLng"
+        :icon="gatePathIcon"
+      >
+        <l-tooltip :options="{ permanent: 'true', direction: 'top' }">Gate Path {{
+        index
+        }}</l-tooltip>
+    </l-marker> 
+
       <!-- Gate Post Icons -->
       <l-marker v-if="post1" :lat-lng="post1" :icon="postIcon">
         <l-tooltip :options="{ permanent: 'true', direction: 'top' }"
```

Comment:
Out of curiosity, why doesn't the full text of these markers seem to show up in the diff? Did we have these in the template already?